Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -581,6 +603,10 @@ theorem mem_rootSet_of_injective [CommRing S] {p : S[X]} [Algebra S R]
   classical
   exact Multiset.mem_toFinset.trans (mem_roots_map_of_injective h hp)
 
+theorem mem_rootSet'' [CommRing S] {p : S[X]} [Algebra S R] [FaithfulSMul S R] {x : R}
```

Comment:
Can you add a docstring describing its difference with the unprimed versions? Thanks.